gdal/build.sh

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
+./configure --with-python --prefix=$PREFIX \
+--with-geos=$PREFIX/bin/geos-config \
+--with-static-proj4=$PREFIX \
+--with-hdf5=$PREFIX \
+--with-hdf4=$PREFIX \
+--with-netcdf=$PREFIX \
+--with-xerces=$PREFIX \
+--with-sqlite=$PREFIX \
+--without-pam \
+--with-python \
```

Comment:
duplicate. Also, you should really be doing `--with-python=${PYTHON}`, I think to guarantee that it is compiled against the right python.